apps/readux/annotations.py

```diff
@@ -6,33 +7,41 @@
 from ..iiif.canvases.models import Canvas
 import json
 import uuid
+from ..users.models import User
+
 
 class Annotations(ListView):
 
-    def get_queryset(self):
-        if self.request.user.is_authenticated:
+    def get_queryset(self, owner):
+        if self.request.user.is_authenticated and owner == self.request.user:
             return UserAnnotation.objects.filter(
-                owner=self.request.user,
+                owner=owner,
                 canvas=Canvas.objects.get(pid=self.kwargs['canvas'])
             )
         return None
 
     def get(self, request, *args, **kwargs):
-        annotations = self.get_queryset()
-        if annotations is not None:
-            for anno in annotations:
-                return JsonResponse(
-                    json.loads(
-                        serialize(
-                            'annotation',
-                            annotations,
-                            # version=kwargs['version'],
-                            is_list = True
-                        )
-                    ),
-                    safe=False,
-                    status=200
-                )
+        username = kwargs['username']
+        try:
+            owner = User.objects.get(username=username)
```

Comment:
Is there a reason to query the database for the user? When we allow people to publish annotations, the query will be needed. Maybe something like:
~~~python
owner = request.user or User.objects.get(username=kwargs['username'])
~~~